packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/platform_interface/platform_interface_document_reference.dart

```diff
@@ -60,20 +58,21 @@ abstract class DocumentReferencePlatform extends PlatformInterface {
 
   /// Deletes the document referred to by this [DocumentReferencePlatform].
   Future<void> delete() {
-    throw UnimplementedError("delete() is not implemented");
+    throw UnimplementedError('delete() is not implemented');
   }
 
   /// Reads the document referenced by this [DocumentReferencePlatform].
   ///
   /// If no document exists, the read will return null.
-  Future<DocumentSnapshotPlatform> get([GetOptions /*?*/ options]) async {
-    throw UnimplementedError("get() is not implemented");
+  Future<DocumentSnapshotPlatform> get(
```

Comment:
You've changed options in the PI here to be non-null, so the following will also need changing:

 -  `Future<QuerySnapshotPlatform> get([GetOptions? options]) async {` in `query_web.dart:123`, options should now be non-null (remove `?`).
 -  `Future<DocumentSnapshotPlatform> get([GetOptions? options]) async {` in `document_reference_web.dart:50`, options should now be non-null (remove `?`).